app/src/main/java/org/p2p/wallet/striga/ui/StrigaOnRampBottomSheet.kt

```diff
@@ -0,0 +1,121 @@
+package org.p2p.wallet.striga.ui
+
+import androidx.fragment.app.FragmentManager
+import android.os.Bundle
+import android.view.View
+import org.p2p.core.common.DrawableContainer
+import org.p2p.core.common.TextContainer
+import org.p2p.uikit.components.finance_block.FinanceBlockCellModel
+import org.p2p.uikit.components.icon_wrapper.IconWrapperCellModel
+import org.p2p.uikit.components.left_side.LeftSideCellModel
+import org.p2p.uikit.components.right_side.RightSideCellModel
+import org.p2p.uikit.utils.drawable.DrawableCellModel
+import org.p2p.uikit.utils.drawable.shape.shapeCircle
+import org.p2p.uikit.utils.drawable.shape.shapeRounded16dp
+import org.p2p.uikit.utils.drawable.shapeDrawable
+import org.p2p.uikit.utils.image.ImageViewCellModel
+import org.p2p.uikit.utils.text.TextViewCellModel
+import org.p2p.uikit.utils.viewState.ViewAccessibilityCellModel
+import org.p2p.wallet.R
+import org.p2p.wallet.common.ui.bottomsheet.BaseBottomSheet
+import org.p2p.wallet.databinding.DialogStrigaOnRampBinding
+import org.p2p.wallet.utils.viewbinding.viewBinding
+
+class StrigaOnRampBottomSheet : BaseBottomSheet(R.layout.dialog_striga_on_ramp) {
+
+    companion object {
+        fun show(fm: FragmentManager) {
+            val tag = StrigaOnRampBottomSheet::javaClass.name
+            if (fm.findFragmentByTag(tag) != null) return
+            StrigaOnRampBottomSheet().show(fm, tag)
+        }
+    }
+
+    private val binding: DialogStrigaOnRampBinding by viewBinding()
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        with(binding) {
+            bankTransferView.bind(
+                model = getFinanceBlock(
+                    titleResId = R.string.striga_bank_transfer_title,
+                    subtitleRes = R.string.striga_bank_transfer_subtitle,
+                    iconResId = R.drawable.ic_striga_bank_transfer,
+                    backgroundTintId = R.color.light_grass
+                )
+            )
+            bankTransferView.setOnClickAction { _, _ ->
+                // TODO PWN-8457
+            }
+            bankCardView.bind(
+                model = getFinanceBlock(
+                    titleResId = R.string.striga_bank_card_title,
```

Comment:
not striga either
just top_up_bank_card_title etc

Reply:
but it's in context of striga feature x D